spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/RetryLoadBalancerInterceptor.java

```diff
@@ -58,6 +58,9 @@
 
 	private final ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory;
 
+	private LoadBalancerPropertiesFactory factoryProperties;
+
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added